README.md

```diff
@@ -42,6 +42,47 @@ pip install matcha-ml
 matcha provision
 ```
 
+Before we start deploying chromadb server on AKS, we need to build the docker image for chromadb server. We build and push this chromadb server image to ACR.
+
+> Note: There exists a [bug](https://github.com/chroma-core/chroma/issues/721) in the exisiting chromadb server image present on [ghcr](https://github.com/chroma-core/chroma/pkgs/container/chroma).
+
+```bash
+acr_registry_uri=$(matcha get container-registry registry-url --output json | sed -n 's/.*"registry-url": "\(.*\)".*/\1/p')
+acr_registry_name=$(matcha get container-registry registry-name --output json | sed -n 's/.*"registry-name": "\(.*\)".*/\1/p')
+```
+
+```bash
+cd infrastructure/chroma_server_k8s
+git clone -b dockerfileChanges --single-branch https://github.com/petersolimine/chroma.git
+cd chroma
+az acr login --name $acr_registry_name
+docker build -t $acr_registry_uri/chroma-server:latest .
+docker push $acr_registry_uri/chroma-server:latest
```

Comment:
I was unable to push the image.

This was the error:
```
docker push $acr_registry_uri/chroma-server:latest
The push refers to repository [crmin53test.azurecr.io/chroma-server]
ceffb986d968: Preparing 
473c8d0a3d90: Preparing 
89db47b2e3c5: Preparing 
5f70bf18a086: Preparing 
c27de61bda54: Preparing 
8c374f3dd48a: Waiting 
4473f86656e8: Waiting 
c43e3868283a: Waiting 
5724726cbe75: Waiting 
8ad0088b4e6d: Waiting 
4b3ba104e9a8: Waiting 
unauthorized: authentication required, visit https://aka.ms/acr/authorization for more information.
```

Reply:
Did the `az acr login --name $acr_registry_name` succeed for you?

Reply:
Yeah.

Reply:
I think so, but there was a warning.

```
You may want to use 'az acr login -n crmin53test --expose-token' to get an access token, which does not require Docker to be installed.
```

Reply:
If it runs correctly, it returns with a message `Login succeeded.`